### ISSUE TITLE
Any user can login influxdb

### ISSUE THREAD

**Issue by feli5** [influxdata/influxdb#7669]
### Bug report

__System info:__ [Include InfluxDB version, operating system name, and other relevant details]
$ influx -version
InfluxDB shell version: 1.1.0
$ cat /etc/redhat-release
CentOS Linux release 7.2.1511 (Core)

__Steps to reproduce:__

1. Enable auth-enabled
2. Restart influxdb
3. Login with any user (Ex: $ influx -username abc -password abc) 

__Expected behavior:__ [Login Fail]

__Actual behavior:__ [Login successfully]

**Comment by e-dard** [comment]
@feli5 can you confirm that you can actually perform queries or writes? When you run `influx` it drops you into the CLI shell, but the auth isn't performed until you actually execute a query/write.

**Comment by feli5** [comment]
@e-dard I'm not using a root user, and I confirmed that I'm able to perform both queries and writes with not exist user.

$ influx -username aa -password aa
Visit https://enterprise.influxdata.com to register for updates, InfluxDB server management, and monitoring.
Connected to http://localhost:8086 version 1.1.0
InfluxDB shell version: 1.1.0
> show users
user		admin
----		-----
influx_admin	true
influxdb_user	true
a		true
admin		true

> show databases
name: databases
name
----
_internal
ocp

> use ocp
Using database ocp
> show measurements
name: measurements
name
----
hypervisor
server

> select last(*) from server
name: server
time	last_addresses			last_hostId							last_id					last_launched			last_name		last_updated
----	--------------			-----------							-------					-------------			---------		------------
0	provider-870:{10.224.243.16}	baca2323ece41431ae281140940235a6ba7d2a199e9dcecd921573f8	322f3e92-4944-48c9-85a6-fdf8d61bcd32	2015-09-02T01:27:52.000000	laisun_test2_private	2016-08-22T13:27:43Z

>
> insert test,host=server02 value=0.67
> select * from test
name: test
time			host		value
----			----		-----
1480518442303176877	server02	0.67

>

**Comment by e-dard** [comment]
@feli5 please provide your configuration file. Are you _sure_ that you're using the config file when you start the server?

**Comment by markusr** [comment]
I was affected by the same strange behaviour. The problem in our case was that is not enough to only uncomment the ` auth-enabled = true ` line. 

You also have to uncomment the `[http]`-line so that it looks like this. 
``` 
[http]
  # Determines whether HTTP endpoint is enabled.
  # enabled = true

  # The bind address used by the HTTP service.
  # bind-address = ":8086"

  # Determines whether HTTP authentication is enabled.
  auth-enabled = true 
```  
After this I was neither able to login nor to do queries.

One of our servers was now running 5 days without password protection (and nobody noticed).

It should however be changed in the shipped conf-file. Or/and abort the start-up if there is a uncommented line in commented section.

**Comment by feli5** [comment]
After I  uncomment the [http]-line, it works. Thanks @markusr and @e-dard .

Agree with markusr, uncommented  section(Ex: [http]) in shipped conf-file would be better.

**Comment by jsternberg** [comment]
I've submitted #7681 to uncomment the lines from the default configuration file. Since the actual configuration file is working when the correct parts are uncommented, I'll count that PR as resolving this issue when it is merged.
